pages/api/genearteReactCodeString.js

```diff
@@ -1,182 +1,15 @@
-
-      if (this.containerStyle) {
-        this.div = `<div style={${parseCSSText(this.containerStyle)}}>${this.html}</div>`;
-      }
-
-      return {
-        html: this.html,
-        div: this.div,
-      };
-    }
-
-    function H2(options) {
-      this.tagName = options.type || null;
-      this.id = options.id || null;
-      this.value = options.value || null;
-      this.style = removeLinebreaks(options.style) || null;
-      this.containerStyle = options.containerStyle || null;
-
-      this.html = `<h2 style={${parseCSSText(this.style)}}>${this.value}</h2>`;
-
-      if (this.containerStyle) {
-        this.div = `<div style={${parseCSSText(this.containerStyle)}}>${this.html}</div>`;
-      }
-
-      return {
-        html: this.html,
-        div: this.div,
-      };
-    }
-
-    function Text(options) {
-      this.tagName = options.type || null;
-      this.id = options.id || null;
-      this.value = options.value || null;
-      this.style = removeLinebreaks(options.style) || null;
-      this.src = options.src || null;
-      this.containerStyle = options.containerStyle || null;
-
-      this.html = `<span style={${parseCSSText(this.style)}}>${this.value}</span>`;
-
-      if (this.containerStyle) {
-        this.div = `<div style={${parseCSSText(this.containerStyle)}}>${this.html}</div>`;
-      }
-
-      return {
-        html: this.html,
-        div: this.div,
-      };
-    }
-
-    // Define a HTMLElementFactory class
-    class HTMLElementFactory {
-      constructor() {}
-      // Factory method for creating elements
-      createElement(options) {
-        switch (options.type) {
-          case 'Button':
-            this.elementClass = Button;
-            break;
-          case 'H1':
-            this.elementClass = H1;
-            break;
-          case 'H2':
-            this.elementClass = H2;
-            break;
-          case 'Image':
-            this.elementClass = Img;
-            break;
-          case 'Input':
-            this.elementClass = Input;
-            break;
-          case 'Text':
-            this.elementClass = Text;
-            break;
-        }
-        // default elementClass is Error
-        return new this.elementClass(options);
-      }
-    }
-
-    // Our default ElementClass is Error object
-    HTMLElementFactory.prototype.elementClass = Error;
-
-    const eleFactory = new HTMLElementFactory();
-
+    
     const parseComponents = (coms, components) => {
       for (const component of coms) {
-        const fullComponentDetails = components.find(c => c.id === component.id)
-        console.log('fullComponentDetails', fullComponentDetails)
-        if (fullComponentDetails) result.push(eleFactory.createElement(fullComponentDetails).div); 
+        const fullComponentDetails = components.find((c) => c.id === component.id);
+        if (fullComponentDetails) result.push(htmlGenerator(fullComponentDetails, framework))
```

Comment:
This style is inconsistent. In this example, the code is all on one line. On line 19, it is in curly braces and on a separate line.